.github/workflows/package-apisix-openresty.yml

```diff
@@ -22,7 +22,7 @@ jobs:
 
       - name: build apisix-openresty rpm
         run: |
-          make package type=rpm app=apisix-openresty version=${BUILD_APISIX_OR_VERSION}
+          make package type=rpm app=apisix-openresty version=${BUILD_APISIX_OR_VERSION} checkout=${BUILD_APISIX_OR_VERSION}
```

Comment:
apisix-openresty uses 'checkout'?

Reply:
Thank you for the reminder. I will remove it.